src/actions/ens/getEnsText.ts

```diff
@@ -20,21 +20,24 @@ import {
   readContract,
 } from '../public/readContract.js'
 
-export type GetEnsTextParameters = Prettify<
+export type GetEnsTextParameters<TKeys> = Prettify<
```

Comment:
Adding constraint + default

```suggestion
export type GetEnsTextParameters<TKeys extends  string | readonly string[] = string | readonly string[]> = Prettify<
```